const-oid/oiddbgen/src/root.rs

```diff
@@ -0,0 +1,90 @@
+use crate::{kind::Kind, node::Node, spec::Spec};
+
+use std::collections::{BTreeMap, HashMap};
+
+use proc_macro2::{Ident, Span, TokenStream};
+use quote::quote;
+
+#[derive(Clone, Debug)]
+pub struct Root {
+    tree: BTreeMap<u8, Kind>,
+    docs: HashMap<u8, (Ident, TokenStream)>,
```

Comment:
I guess this makes sense, but looks a little weird

Reply:
What looks weird?

Reply:
Mostly just using `BTreeMap` for one and `HashMap` for the other

Reply:
This is by design. The sorting in the `BTreeMap` makes the output predicable. OTOH, the `HashMap` makes the lookups faster.

Reply:
It's mostly a moot point, but FWIW the performance seems identical with `BTreeMap` in place of `HashMap`